_docs/Qabel-Protocol-Box.md

```diff
@@ -281,6 +285,19 @@ url: STR // URL to the metadata file that contains information about the folder
 },
 ```
 
+ExternalFile:
+
+```
+{
+owner: STR, // public key of the owner of that File
+name: STR, // object name,
+size: LONG, // uncompressed file size
+mtime: LONG, // modification time as seconds since epoch
```

Comment:
I guess one would have to refresh the external files regularly and update these information. Additionally, this information would only reach the FM if the creator of the share would update them regularly.